pkg/system/phase2_creating.go

```diff
@@ -154,6 +154,7 @@ func (r *Reconciler) SetDesiredNooBaaDB() {
 	if r.NooBaaDB.UID == "" {
 		for i := range r.NooBaaDB.Spec.VolumeClaimTemplates {
 			pvc := &r.NooBaaDB.Spec.VolumeClaimTemplates[i]
+			pvc.Namespace = r.NooBaaDB.Namespace
```

Comment:
Is this change related?

Reply:
it failes the next step in the namespace isn't set - so yes